src/i18n/es.json

```diff
@@ -115,6 +115,7 @@
     "title": "Solicita una actualización de tu plan"
   },
   "validation_messages": {
+    "error_account_is_canceled" : "Esta cuenta se encuentra cancelada. Por favor, contacta a soporte.",
```

Comment:
@carodipietro could you review this content?

Reply:
By the moment I will apply the same that we are using in now.

Reply:
Este contenido no está generado para esta versión. Podés usar lo que está en Doppler actualmente hasta que se generen los nuevos contenidos.